packages/commerce-sdk-react/README.md

```diff
@@ -26,17 +26,23 @@ const AppConfig = ({children}) => {
     )
 } 
 
+// Custom retry logic
+// Ref: https://tanstack.com/query/v4/docs/guides/query-retries
+const shouldRetry = (failureCount, error) => {
+    // For example: do not retry on server side, but keep retrying until max of 3 failures on the client-side
+    return isServer ? false : failureCount < 3
+}
+
 // Set configuration options for react query.
-// NOTE: This configuration will be used both on the server-side and 
-// client-side.
+// NOTE: This configuration will be used both on the server-side and client-side.
```

Comment:
@adamraya I think our example could have a simple option `retry: false`. But then I saw this comment here that Ben wrote. So that's why I changed my mind and decided to align with Ben's original intention.